core/ui.js

```diff
@@ -213,7 +215,8 @@ async function addIconIfNeeded()
 
 function generateDropContent(options)
 {
-    var presenceCaption = "Will prevent you from seeing other people's last seen and typing status";
+    var onlineStatusCaption = "Will prevent you from being online. If you enable this, you won't be able to see others' status.";
```

Comment:
"Prevent you from being online" sounds a bit weird. Maybe we can write "Stops sending presence updates. If enabled, will prevent you from seeing other's online status"